from __future__ import annotations


class RpcException(Exception):
    pass

